### PR TITLE
Added handling for YAML config files.

### PR DESCRIPTION
**Pull request by payammeyer**

### CODE REVIEW

File: zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java [hunk 1]
```diff
@@ -50,7 +52,13 @@ public ZuliaDConfig(String config) throws IOException {
 
 		LOG.info("Loading config <" + config + ">");
 
-		zuliaConfig = GSON.fromJson(new FileReader(config), ZuliaConfig.class);
+		if (config.endsWith("json") || config.endsWith("properties")) {
```
Comment (mdavis95) [inline zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java:55]: should we do config.toLower here?

File: zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java [hunk 2]
```diff
@@ -50,7 +52,13 @@ public ZuliaDConfig(String config) throws IOException {
...
+			zuliaConfig = GSON.fromJson(new FileReader(config), ZuliaConfig.class);
+		}
+		else if (config.endsWith("yml") || config.endsWith("yaml")) {
+			Yaml yaml = new Yaml(new Constructor(ZuliaConfig.class));
+			zuliaConfig = yaml.load(new FileReader(config));
```
Comment (mdavis95) [inline zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java:60]: Do we need to close the file reader (in json and yaml)?

File: zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java (hunk 1 above)
Comment (payammeyer) [reply zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java:55]: I mean... we could but I would say capital filenames are wrong.

File: zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java (hunk 2 above)
Comment (payammeyer) [reply zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java:60]: I'm not seeing a warning, so maybe not?

**Pull request merged**